### ISSUE TITLE
Can't install into Rails 3 project

### ISSUE THREAD

**Issue by jamie** [clio/ten_years_rails#15]
I'm trying to add this to a Rails 3 project to start a Rails 4 upgrade, and ignoring #9 it won't even bundle. It seems `ten_years_rails` says it requires `actionview` for something, and that was only introduced as a standalone gem with rails 4 so the install fails:

```
Bundler could not find compatible versions for gem "activesupport":
  In snapshot (Gemfile.lock):
    activesupport (= 3.2.22.13)

  In Gemfile:
    ten_years_rails was resolved to 0.2.0, which depends on
      actionview was resolved to 4.1.0.beta1, which depends on
        activesupport (= 4.1.0.beta1)

    rails (~> 3.2.22.13) was resolved to 3.2.22.13, which depends on
      activesupport (= 3.2.22.13)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by etagwerker** [comment]
@jamie I think it might be solved in #14. Could you test with that branch? 

Also, just curious, what version of Ruby are you using?

**Comment by jamie** [comment]
I think that #14 doesn't go far enough, it's still got `spec.add_dependency "actionview"` which will conflict dependencies with Rails 3 eventually. I'll give it a shot this afternoon though.

Currently we're running ruby 2.1, but I want to push for a 2.2 upgrade since that will give us a clear runway through to rails 5.2.

**Comment by jamie** [comment]
@etagwerker yeah, loading it as `gem 'ten_years_rails', github: 'fastruby/ten_years_rails_conf_2018', branch: 'fixes/ruby-2-2-syntax-error'` gives me the same thing, actionview pulling in a conflicting activesupport version.

**Comment by etagwerker** [comment]
@jamie I guess there are two ways to install `ten_years_rails`, using your `Gemfile` and `bundler` or simply with `gem install ten_years_rails` -- have you tried with `gem install ten_years_rails`?

**Comment by stoem** [comment]
Just warming this up, I used `gem install ten_years_rails` in a Rails 3 app and it gave me 

> You have already activated activesupport 5.2.8.1, but your Gemfile requires activesupport 3.2.22.29. Prepending `bundle exec` to your command may solve this.

I tried with bundle exec and got 

> can't find executable bundle_report for gem ten_years_rails. ten_years_rails is not currently included in the bundle, perhaps you meant to add it to your Gemfile?

Any tips?

**Comment by etagwerker** [comment]
@stoem There's a gem called next_rails that doesn't have the dependency on activesupport. You might wanna give it a try. 

The other alternative is to add ten_years_rails to your gemfile, bundle, and try again.

**Comment by stoem** [comment]
@etagwerker Thank you, go it. That works for me.
